src/lib.rs

```diff
@@ -1,51 +1,83 @@
-use std::{fs, time::SystemTime};
+use std::{fmt::Display, fs, time::SystemTime};
 
 use chrono::{DateTime, Utc};
+use futures::{stream, StreamExt};
 use itertools::Itertools;
 use lazy_static::lazy_static;
 use log::{info, warn};
 use regex::Regex;
+use reqwest::Client;
 use std::fmt::Write as _;
+use tokio::runtime::Handle;
 
 pub use blog::{Blog, Post};
 mod blog;
 mod xml;
 
 use crate::xml::parse_web_feed;
 
+const CONCURRENT_REQUESTS: usize = 10;
```

Comment:
What was the thought process for arriving at this value? Were other values considered?

After trying `5` and `15`, `10` seems to be working better on my machine, just curious.

Reply:
Yes, I did try various sizes, 10 seemed a good in between, lower were getting me slower times and higher would risk having the problem you had on Windows when all were concurrent.

Reply:
Makes sense